obspy/core/inventory/util.py

```diff
@@ -963,6 +963,59 @@ def _unified_content_strings(contents):
     return items
 
 
+def _unified_content_strings_expanded(contents):
+    contents2 = [["." + item.location_code, item.code,
+                  item.sample_rate, item.start_date, item.end_date,
+                  item.depth]
+                 for item in contents]
+
+    # sorting this to keep the channels Z-N-E
+    contents2 = sorted(contents2, key=lambda x: (x[2], x[1], x[3]),
+                       reverse=True)
```

Comment:
Not sure what is in `contents2` here, but maybe we should think about a custom sort routine for component code?

Something like sort `ZNE0123456789` and then everything else a-Z? `reverse` is a quick and dirty crutch for `ZNE` I use too sometimes but now numbers are comin in backwards and for other characters it might look weird too.